Products/Products-Root/Screens/Authentication/Login/LoginViewController.swift

```diff
@@ -39,6 +42,21 @@ class LoginViewController: BaseViewController, AlertPresenter, LoadingViewPresen
     @IBAction func loginButtonPressed(_ sender: Any) {
         viewModel.login()
     }
+    
+    @IBAction func forgotPasswordButtonPressed(_ sender: Any) {
+    }
+    
+    @IBAction func showPasswordButtonPressed(_ sender: Any) {
+        if (passwordHidden == true) {
+            showPasswordButton.setTitle("Hide", for: .normal)
```

Comment:
please add this "Hide" string to the Localizable.strings file, for the key use something like "hide" and write "hide".localized here